plugins/qtAccessionPlugin/modules/accession/templates/browseSuccess.php

```diff
@@ -11,7 +11,8 @@
     <div class="row">
       <div class="span6">
         <?php echo get_component('search', 'inlineSearch', [
-            'label' => __('Search %1%', ['%1%' => strtolower(sfConfig::get('app_ui_label_accession'))]), ]); ?>
+            'label' => __('Search accessions'),
+            'landmarkLabel' => __('Accession'), ]); ?>
```

Comment:
This is something that has been puzzling me for a while and I'd like to get @fiver-watson input on it. There are a few places where we need to use the entities' UI labels in plural, like this one. Considering, i18n this is not something you can do manually so we end up not using those labels or using them in singular.

We could add another UI label setting for each entity to include a plural version. I'd consider it another issue if Dan likes the idea, but I'd keep using the UI labels in these cases until we address that one.

Reply:
Just to clarify, I removed the use of UI labels for 'accession' and 'donor' because, as far as I can tell, these labels were never actually created. I made a note of it and was thinking about adding them in a different PR. I also noticed the plural issue, but I can't immediately think of a better solution than what you've suggested.

Reply:
Hey both, 

I do think that any changes around UI labels and plurality should probably be addressed in a separate PR/issue, so the work isn't lost in here, and/or blooms into something much larger that no longer fits the original PR purpose. 

This has certainly been an issue. In the past it's been dealt with on a case by case basis, so the solutions aren't always consistent across entity types. Mainly I think we need to establish a clear solution that can work for all the various page types and headers, and then focus on implementing it consistently going forward, rather than trying to sneak in a fix or two in different places, as this is part of what's led us to some inconsistency across entities. I'll add a few initial thoughts below, however. 

I do think there's a certain amount of mileage we can get from simply structuring strings a bit differently - for example, "Donor browse", "Authority record browse" etc could potentially work fine for most entities, but is not currently the norm. In some cases, adding "records" could solve a lot of issues around plurality (Browse donor records, browse repository records, browse archival description records, etc).... but it's a bit ungainly, and more importantly, it wouldn't work for authority records (Browse authority record records, lol).

Adding a whole set of separate label UI elements for plural cases would be nice, but also a fair bit of work, and likely needs consideration in terms of both ongoing maintenance, and impact on upgrading users (not sure how translations, fixtures, etc will factor into all of this for example). 

All this to say: I think it makes sense to address this as a separate issue, and inform any decision by trying to gather up some concrete examples of different places where we need to consider plurality.